src/sentry/static/sentry/app/components/version.tsx

```diff
@@ -1,7 +1,10 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import styled from '@emotion/styled';
+import * as ReactRouter from 'react-router';
```

Comment:
Wouldn't it be better to export only what you really need?

Reply:
Yes, but the thing is, we can't do:
```
import {withRouter, WithRouterProps} from 'react-router';
```

we could do:
```
import {withRouter} from 'react-router';
import {WithRouterProps} from 'react-router/lib/withRouter';
```

I've seen `import * as ReactRouter from 'react-router';` syntax used in the codebase, so I went with that option.

Do you like the second one more?

Reply:
I would go for the second one :)

Reply:
```suggestion
import {withRouter} from 'react-router';
import {WithRouterProps} from 'react-router/lib/withRouter';
```